todo-app/src/components/addingContainer.js

```diff
@@ -0,0 +1,43 @@
+
+import React, { Component } from "react";
+import FaPlus from "react-icons/lib/fa/plus"
+
+
+export default class AddingContainer extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            description: "",
+            deadLine: ""
+        }
+    }
+
+    handleDescriptionChange = (event) => {
+        this.setState({ description: event.target.value })
+    }
+    handleDeadlineChange = (event) => {
+        this.setState({ deadLine: event.target.value })
+
+    }
+    handleCreating = () => {
+        let descriptionInput = this.state.description
+        let deadLineInput = this.state.deadLine
+         this.props.create(descriptionInput, deadLineInput)
+    }
+
+    render() {
+        let description = <input type="text" value={this.state.description} onChange={this.handleDescriptionChange} />;
+        let deadLine = <input type="text" value={this.state.deadLine} onChange={this.handleDeadlineChange} />;
```

Comment:
So, you're already well beyond week 1's homework! Good job.

At the moment, your deadline input is a text input, and I can type anything into it at all. Here's an example that doesn't make much sense:

![screen shot 2018-03-05 at 11 21 32](https://user-images.githubusercontent.com/10575782/36970017-6404d6fe-2067-11e8-828d-6da450bb610f.png)

Let's think about what these deadlines are in the first place - they're dates/date objects, right? I'd like to see you use either a native HTML5 date input, or a third party library (pulled in from `npm`) to allow users to easily select dates from a calendar UI.